README.md

```diff
@@ -28,9 +27,15 @@ which it's run:
 - `modprobe`
 - `gpg`
 
+### Prebuilt Binaries
+
+The easiest way to get `acbuild` is to download one of the
+[releases](https://github.com/appc/acbuild/releases) from GitHub.
+
 ### Build from source
 
-Currently the only way to install `acbuild` is to build from source. 
+The other way to get `acbuild` is to build it from source.
```

Comment:
Might mention the platform supported/required, which far as I can tell is only Linux amd64 (right? --at any rate I can't build it on darwin)